### PR TITLE
Feature/maxon epos ethercat sdk

### PR DESCRIPTION
**Pull request by zoenglinghou**

### CODE REVIEW

**Comment by firesurfer** [comment]
@jungerj This PR doesnt build because of the added sdo read methods. Do you really need those methods? I cant really imagine that. Otherwise make sure that you merge them before.
EDIT: Sorry this comment should actually go to the maxon ethercat sdk

**Comment by zoenglinghou** [comment]
well it seems that the failure to the **configurator** is that it is missing some dependencies?

**Comment by zoenglinghou** [comment]
The new `Pdo` type is just like `PdoStandard`. We create this type to accommodate the objects required for the `Pdo`s when using multiple modes. Currently we only implemented a mode for CST/CSP, but more would be added should other modes are desired.

File: config/device_configurations/maxon.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Maxon:
```
Comment (firesurfer) [inline config/device_configurations/maxon.yaml]: Why did you create an config folder? This should not be in here. Please add a maxon yaml in the examle config. 
For your specific setup you should then have an extra wrist package that contains your config.

**Comment by firesurfer** [comment]
@zoenglinghou see my comment on the configuration stuff. Apart from that the PR is mergable I would say

File: config/device_configurations/maxon.yaml (hunk 1 above)
Comment (zoenglinghou) [reply config/device_configurations/maxon.yaml]: You are right... I created for my own config, I will remove it.

**Comment by jungerj** [comment]
pulled in the last changes from master (rokubimini_ethercat_sdk)

**Comment by zoenglinghou** [comment]
Is there any more changes required?

**Pull request merged**